tests/parity/test_active_suite.py

```
@@ -0,0 +1,545 @@
+            "model_resolution": "canonicalized",
+        },
+        model="claude-fable-5",
+    ) is False
+
+
+@pytest.mark.parametrize(
+    ("requested_model", "canonical_model"),
+    (
+        ("claude-fable-5", "claude-unapproved"),
+        ("claude-fable-4", "claude-fable-5-1"),
+    ),
```

Ruff (0.16.3)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)